react/components/atoms/button/button.js

```diff
@@ -3,24 +3,28 @@ import { ActivityIndicator, Platform, StyleSheet, Text, View, ViewPropTypes } fr
 import LinearGradient from "react-native-linear-gradient";
 import PropTypes from "prop-types";
 import { mix } from "yonius";
+import { Icon, Touchable } from "ripe-components-react-native";

```

Comment:
Importing stuff from itself? This is kind of weird, was it a typo?

Reply:
Ah, typo!